scripts/apps/archive/views/preview.html

```diff
@@ -48,11 +48,11 @@
             </div>
             <div class="tag-main" ng-if="selected.preview.authors">
               <span class="inline-label">{{preview_headings.author}}:</span>
-              <span class="tag-label" ng-repeat="author in selected.preview.authors">{{author.role}}:{{author.sub_label}}</span>
+              <span class="tag-label" ng-repeat="author in selected.preview.authors" title="{{author.role}}:{{author.sub_label}}">{{author.role}}:{{author.sub_label}}</span>
             </div>
             <div class="tag-main" ng-if="selected.preview.keywords">
               <span class="inline-label"><br/>{{preview_headings.keywords}}:</span>
-              <span class="tag-label" ng-repeat="keyword in selected.preview.keywords" title="Keywords">{{keyword}}</span>
+              <span class="tag-label" ng-repeat="keyword in selected.preview.keywords" title="{{keyword}}">{{keyword}}</span>
```

Comment:
I think in general we don't need title for span if it's the same as the actual value, not sure if we use it anywhere else?

Reply:
actually, I'm added it to maintain the consistency like we have a title for subject-related fields also(here it is https://github.com/superdesk/superdesk-client-core/blob/36822018045b10c9d8c838d34a6bd6c6b4d106f5/scripts/apps/search/components/preview-subject.tsx#L38) so that while content-preview we have a title for author and storytags like we have for subjects(Belga-keywords etc)